state/Readme.md

```diff
@@ -11,6 +11,7 @@ Currently supported state stores are:
 * Azure CosmosDB
 * Memcached
 * Zookeeper
+* Cloud Firestore
```

Comment:
Can we change this to `Cloud Firestore (Datastore mode)`?

Reply:
@yaron2  Sure, I can